tsconfig.json

```diff
@@ -9,7 +9,7 @@
         "exchanges/*/src"
       ],
       "@urql/core/*": ["node_modules/@urql/core/src/*", "packages/core/src/*"],
-      "@urql/devtools": ["node_modules/@urql/devtools"],
+      "@urql/devtools": ["node_modules/@urql/devtools", "exchanges/devtools/src"],
```

Comment:
I'm not sure why this entry was already here, haha, just wondering if it's needed given the below entry, since we'd avoid conflicts by removing it